src/PrivateKey.cpp

```diff
@@ -145,6 +146,25 @@ PublicKey PrivateKey::getPublicKey(TWPublicKeyType type) const {
     return PublicKey(result, type);
 }
 
+Data PrivateKey::getSharedKey(const PublicKey& pubKey, TWCurve curve) const {
+    if (curve != TWCurveSECP256k1) {
+        return {};
+    }
+
+    Data result;
+    result.resize(65);
+    bool success = ecdh_multiply(&secp256k1, bytes.data(),
+                                 pubKey.bytes.data(), result.data()) == 0;
+
+    if (!success) {
+        return {};
```

Comment:
Trezor ecdh_multiply() never returns error, so error check here cannot be covered by test... :(   For the sake of coverage, I suggest reverting the if
```
    if (success) {
        ...
    }
```